pypeit/core/flux_calib.py

```diff
@@ -548,10 +549,81 @@ def sensfunc(wave, counts, counts_ivar, counts_mask, exptime, airmass, std_dict,
 
     return meta_table, out_table
 
+def get_sensfunc_factor(wave, wave_sens, sensfunc, exptime, telluric=None, extinct_correct=False,
+                         airmass=None, longitude=None, latitude=None, extrap_sens=False):
+    """
+    Get sensitivity function factor that will be multiplied into a spectrum. This code interpolates the sensitivity
+    function and can also multiply in extinction and telluric corrections.
+
+    FLAM, FLAM_SIG, and FLAM_IVAR are generated
+
+    Args:
+        wave (float ndarray): shape = (nspec,)
+           Senstivity
+        wave_sens (float ndarray):
+           Sensfunc wavelength vector shape = (nsens,)
+        sensfunc (float ndarray): shape = (nsens,)
+           Sensitivity function
+        exptime (float):
+        telluric_correct:
```

Comment:
change to `telluric (float ndarray, optional): shape = (nspec,)`

Reply:
Changed.